onnxruntime/test/providers/cpu/model_tests.cc

```diff
@@ -108,13 +108,13 @@ TEST_P(ModelTest, Run) {
   }
 
   std::unique_ptr<OnnxModelInfo> model_info = std::make_unique<OnnxModelInfo>(model_path.c_str());
-  if (model_info->GetONNXOpSetVersion() != 14 && model_info->GetONNXOpSetVersion() != 15 &&
+  if (model_info->GetONNXOpSetVersion() < 14 || model_info->GetONNXOpSetVersion() > 17 &&
```

Comment:
(model_info->GetONNXOpSetVersion() < 14 || model_info->GetONNXOpSetVersion() > 17) && provider_name == "tensorrt"

Reply:
Thanks for the call out. Will be fixed soon